interceptors/logging/logging.go

```diff
@@ -47,12 +60,12 @@ func DefaultErrorToCode(err error) codes.Code {
 }
 
 // Decider function defines rules for suppressing any interceptor logs
-type Decider func(fullMethodName string, err error) bool
+type Decider func(fullMethodName string) Decision
```

Comment:
Would have been nice not to have skipped the error parameter here. The logging can also be configured using the error as well,(since status code can be derived from the response error).